tool/dependencies/known-dependencies.txt

```diff
@@ -705,4 +705,16 @@ kerb-simplekdc-1.0.1.jar
 kerb-util-1.0.1.jar
 kerby-asn1-1.0.1.jar
 kerby-config-1.0.1.jar
-kerby-pkix-1.0.1.jar
\ No newline at end of file
+kerby-pkix-1.0.1.jar
+hadoop-aliyun-3.3.4.jar
+aliyun-sdk-oss-3.16.0.jar
+aliyun-java-sdk-core-4.5.10.jar
```

Comment:
need to add the license to dir: linkis-dist/release-docs/licenses, And the protocol needs to be compatible with apache v2

Reply:
Ok ,I have add new LICENSE  to dir: linkis-dist/release-docs/licenses. Please correct me if I was wrong .Thanks for your guide.

Reply:
> Ok ,I have add new LICENSEs to dir: linkis-dist/release-docs/licenses. Please correct me if I was wrong .Thanks for your guide.

Thank you, Got it